test/torch_np/numpy_tests/core/test_numeric.py

```diff
@@ -232,14 +232,14 @@ def test_dunder_round(self, dtype):
             # pytest.param(
             #    2**31 - 1, -1, marks=pytest.mark.xfail(reason="Out of range of int32")
             # ),
-            subtest((2**31 - 1, -1), decorators=[xfail]),
+            subtest((2**31 - 1, -1), decorators=[xfailIfTorchDynamo]),
```

Comment:
I think the failing cases are when not run with dynamo, so keeping the xfail should make them pass? Were you seeing something different?

Reply:
Nope, the regular `xfail` also gives an unexpected success: https://hud.pytorch.org/pytorch/pytorch/pull/115259?sha=df29bbecf28ffdde8b2cbd68cd17ee373c9f6438. Gonna try to xfail when its not dynamo now. In retrospect that makes more sense. If we replace the numpy scalar with a tensor and the latter is now supported in dynamo, this should pass.

Reply:
yep, xfailing eager makes sense here as discussed offline